### PR TITLE
master-lps-112548

### PR DESCRIPTION
**Pull request by liuxingye**
https://issues.liferay.com/browse/LPS-112548

### CODE REVIEW

**Comment by liuxingye** [comment]
ci:test:sf

**Comment by austinchiang** [comment]
**Reviewing...** :hourglass_flowing_sand: :sunglasses: :headphones:

File: modules/apps/depot/depot-test/src/testFunctional/tests/DepotWorkflow.testcase [hunk 1]
```diff
@@ -321,6 +321,55 @@ definition {
 			webContentTitle = "Depot WebContent Title");
 	}
 
+	@description = "This ensures that a single approver workflow of web content can be defined for the asset library."
+	@priority = "5"
+	test ApproveWCArticleWithinAL {
+		property custom.properties = "jsonws.web.service.paths.excludes=";
```
Comment (austinchiang) [inline modules/apps/depot/depot-test/src/testFunctional/tests/DepotWorkflow.testcase:327]: Is this line necessary? I don't see any JSON requests being made in the test

File: modules/apps/depot/depot-test/src/testFunctional/tests/DepotWorkflow.testcase [hunk 2]
```diff
@@ -665,6 +714,181 @@ definition {
 		PortletEntry.submitForPublication();
 	}
 
+	@description = "This ensures that the resubmitted web content can be approved in an asset library."
+	@priority = "5"
+	test RejectandResubmitWCArticleWithinAL {
+		property custom.properties = "jsonws.web.service.paths.excludes=";
```
Comment (austinchiang) [inline modules/apps/depot/depot-test/src/testFunctional/tests/DepotWorkflow.testcase:720]: Same here

**Comment by austinchiang** [comment]
I'll push this in for now:
https://github.com/liferay-lima/liferay-portal/pull/335

**Pull request closed without merging**